src/classes/BDI_MappingServiceAdvanced.cls

```diff
@@ -174,7 +195,7 @@ public with sharing class BDI_MappingServiceAdvanced implements BDI_MappingServi
         }
     }
 
-    private void retrieveCustMetadata(){
+    private void retrieveCustMetadata(Boolean includeUnmappedFields) {
```

Comment:
is "Unmapped fields" the best name? There are other fields on DI that aren't mapped that are still left out. Maybe utilityFields? Not a critical decision.